project_event/models/project_task.py

```diff
@@ -175,6 +176,28 @@ def _onchange_resource_type(self):
         self.room_id = None
         self.equipment_id = None
 
+    @api.onchange('room_id')
+    def verify_room_bookable(self):
+        if self.room_id:
```

Comment:
@eilstWhat is the validation message "if self.room_id; " condition fails Or if this condition is going to true always what is the use of checking this condition.

Reply:
this is a condition to catch the error, so the else condition is implicit, to pass